javascript-modal/main.js

```diff
@@ -0,0 +1,28 @@
+// create storage for directory to the modal box
+// create storage for directory to the open button
+// create action to take when a user clicks the open button
+// when open button clicks
+//      - if the modal box is not visible, make modal box appear
+// make storage for directory to no button
+// create action to take when a user clicks the no button
+// when no button clicks
+//       - hide modal
+
+var hidden = document.querySelector('div.relative');
+var openButton = document.querySelector('button.open-button');
+openButton.addEventListener('click', modalStatus);
+
+function modalStatus(event) {
+  if (hidden.className === 'relative display-none') {
```

Comment:
If this function runs, then the element will be hidden, since the purpose of this function is to show the modal. There's no conditional necessary here, you can just do the work.